src/rtk/features/overviewOrg/overviewOrgSlice.ts

```diff
@@ -0,0 +1,246 @@
+import axios from 'axios';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+import { DeleteOrgDto } from '@/shared/dto/DeleteOrgDto';
+
+export interface ApiListItem {
+  id: string;
+  name: string;
+  isSoftDelete: boolean;
+  owner: string;
+  description: string;
+}
+
+export interface OverviewOrg {
+  id: string;
+  name: string;
+  orgNr: string;
+  isAllSoftDeleted: boolean;
+  apiList: ApiListItem[];
+}
+
+export interface InitialState {
+  loading: boolean;
+  overviewOrgs: OverviewOrg[];
+  softDeletedOverviewOrgs: OverviewOrg[];
```

Comment:
Add something else than softDeletedOverviewOrgs to count if there exists softDeleted items. softDeletedItems counter or something.